### ISSUE TITLE
introducing the setting

### ISSUE THREAD

**Issue by rorywhite200** [joelostblom/viz-oer#35]
I think we may want to have a paragraph introducing the setting / Viridia at the very beginning of the text - perhaps even in the section of how to use this book.

**Comment by joelostblom** [comment]
I agree that we should have an introduction to the world somewhere early on

**Comment by joelostblom** [comment]
We talked about this today and agreed to try introducing it in one paragraph at the beginning of chapter one. We will also introduce one exercise right after that paragraph that makes it clear how the reader has to learn about data visualization to help the Viridians, before we get into the content of that chapter.

We are still undecided on whether the Viridians will be references throughout all chapter text or mostly around the exercises.

**Comment by joelostblom** [comment]
Done in #61
